internal/component/database_observability/mysql/gatherer.go

```diff
@@ -0,0 +1,85 @@
+package mysql
```

Comment:
This can be shared with `database_observability.postgres`.

Reply:
I was just thinking that, this can be shared between the two components